tool/cli.js

```diff
@@ -46,14 +46,36 @@ program
   .name("tool")
   .version("0.0.0")
   .disableGlobalOption("--silent")
-  .command("validate-redirects", "Check the _redirects.txt file(s)")
+  .command("validate-redirects", "Try loading the _redirects.txt file(s)")
   .action(
     tryOrExit(({ logger }) => {
       Redirect.load(null, true);
       logger.info(chalk.green("🍾 All is well in the world of redirects 🥂"));
     })
   )
 
+  .command("verify-redirects", "Verify the _redirects.txt file(s) by locale")
```

Comment:
Happy to rename. That's what we should run on PRs for `content` and `translated-content`